app/routes/bank/edit/$citationId.tsx

```diff
@@ -75,7 +74,7 @@ export const action = async ({ request, params }: ActionArgs) => {
       : 0,
   }
 
-  const updatedCitation = await updateCitation(citationFields)
+  await updateCitation(citationFields)
```

Comment:
Not sure if it's here, but when you try to update a citation for which there is no headword / the headword doesn't exist, the error isn't caught

Reply:
like for the entries for "b" for example

Reply:
Yeah there's a whole lotta exception handling that needs doing 😅

Reply:
That said, I can't seem to reproduce the error. The new headword should be created by the call to `findOrCreateHeadword`. I'm going to close this conversation, but file a new issue for this.